packages/demo/package.json

```diff
@@ -29,32 +29,39 @@
   },
   "homepage": "https://github.com/apache-superset/superset-ui#readme",
   "dependencies": {
-    "@storybook/addon-actions": "^5.0.6",
-    "@storybook/addon-knobs": "^5.2.3",
-    "@storybook/addon-links": "^5.0.6",
-    "@storybook/addons": "^5.0.6",
-    "@storybook/react": "^5.2.3",
-    "@superset-ui/chart": "0.12.12",
-    "@superset-ui/color": "0.12.12",
-    "@superset-ui/connection": "0.12.12",
+    "@storybook/addon-info": "^5.3.18",
+    "@storybook/addon-actions": "^5.3.18",
+    "@storybook/addon-knobs": "^5.3.18",
+    "@storybook/addon-links": "^5.3.18",
+    "@storybook/addons": "^5.3.18",
+    "@storybook/react": "^5.3.18",
+    "@storybook/preset-typescript": "^3.0.0",
+    "@superset-ui/chart": "*",
+    "@superset-ui/color": "*",
+    "@superset-ui/connection": "*",
     "@superset-ui/legacy-plugin-chart-sankey": "^0.11.15",
     "@superset-ui/legacy-plugin-chart-sunburst": "^0.11.15",
     "@superset-ui/legacy-plugin-chart-word-cloud": "^0.11.15",
-    "@superset-ui/legacy-preset-chart-big-number": "^0.11.15",
-    "@superset-ui/number-format": "0.12.12",
+    "@superset-ui/legacy-plugin-chart-table": "*",
+    "@superset-ui/legacy-preset-chart-big-number": "*",
+    "@superset-ui/number-format": "*",
     "@superset-ui/plugin-chart-word-cloud": "^0.11.15",
-    "@superset-ui/query": "0.12.12",
-    "@superset-ui/time-format": "0.12.12",
+    "@superset-ui/query": "*",
+    "@superset-ui/time-format": "*",
+    "@superset-ui/translation": "*",
```

Comment:
The demo package should always be built within the monorepo, which always has other Superset packages as symlinks----thanks to [yarn workspaces](https://classic.yarnpkg.com/en/docs/workspaces/). Setting their versions to `*` so we don't have to manually keep them in sync.

Other packages used in the demo but have not migrated to this monorepo do have a specified version, because they need to be installed from npm.